### DISCUSSION TITLE
Could we use $true for a [bool] parameter instead of '1'?

### DISCUSSION THREAD

**Discussion by alexandair** [PowerShell/PSArm#149] [Q&A]
Looking at the examples, I've noticed this:

```powershell
[Parameter()] [bool] $supportsHTTPSTrafficOnly = 1,
```

I've learnt from @rjmholt, that `1` comes from the ARM schema.
PowerShell users would expect `$true`. Is there a way to convert `$true` to `1` behind the scenes?

**Comment by vexx32** [comment]
PS would convert that value to `$true` anyway because the parameter is `[bool]`, so there's not really any actual value in defining the parameter like that.
